### PR TITLE
docs: Added syncset and seletorsyncset to FAQ.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @dgoodwin

### CODE REVIEW

File: docs/FAQs.md [hunk 1]
```diff
@@ -18,3 +25,9 @@ However this effort had to be abandoned due to some conflicts in how Hive segreg
 ## How does Hive relate to the sig-cluster-lifecycle Cluster API project?
 
 Hive is built to leverage the opinionated OpenShift 4 install process. This does not presently have any overlap with the ClusterAPI actuators, although OpenShift 4 itself does leverage a subset of the Cluster API in-cluster with use of the Machine API.
+
+## Why not merge `SelectorSyncSet` and `SyncSet` into one CRD?
+
+`SyncSets` transfer per cluster certificates, identity providers, lists of dedicated admin usernames, etc. Pushing this up to a global `SelectorSyncSet` CRD complicates RBAC and potentially exposes us to reveal more information to someone than we wanted to. The distinction between the two offers better flexibility for RBAC in a multi-tenant use of Hive, and possibly better security as well.
+
+Pleaes refer to [Issue 601](https://github.com/openshift/hive/issues/601) to get more detail.
```
Comment (dgoodwin) [inline docs/FAQs.md]: s/Pleaes/Please, otherwise looks good to me. Thanks!

Comment (gyliu513) [reply docs/FAQs.md]: Done, thanks @dgoodwin

**Pull request merged**